### DISCUSSION TITLE
How to get length?

### DISCUSSION THREAD

**Discussion by Anemoiaa** [mozman/ezdxf#829] [Q&A]
Need  get length like this:
![image](https://user-images.githubusercontent.com/99463624/217470903-1df509e6-fee8-403b-a05b-37c57ee47c58.png)

**Comment by catalinanesia** [comment]
Hei @Anemoiaa
It is cool to ask quick questions on a topic but NOT/cool to be so blunt...
(no research on your part)
You can achieve all that via ezdxf library
**FIRST** (DYOR) - a lot of that

1) - convert the CorelDraw to dxf
2) - write the python script to parse the DXF file for entity's
3) - extract the coordinate points for all the objects
4) - use math to calculate what you need starting with the extracted points coordinates

Optional - get a programmer
